### ISSUE TITLE
Bot crashes

### ISSUE THREAD

**Issue by kunalvsh** [llimllib/limbo#48]
I've been getting this error occasionally with the bot crashing after a couple of days:

![screen shot 2015-07-29 at 1 27 47 pm](https://cloud.githubusercontent.com/assets/2515263/8969021/a3b1fa84-35f5-11e5-8008-624407a94f6c.png)

Any reason why or how to fix this?

**Comment by llimllib** [comment]
Shoot, sorry I've fixed this on my development branch but have fallen off on working on this project, so I haven't made a release.

I will try to make a release tonight or tomorrow! If you want, you can run off the `slackrtm` branch, or you can change lines 98-103 to:

```
    try:
        msguser = server.slack.server.users[event["user"]]
    except KeyError:
        logger.debug("event {0} has no user".format(event))
        return
```

So sorry about that!

**Comment by kunalvsh** [comment]
No worries! Thanks so much for the super quick response.

**Comment by kunalvsh** [comment]
So I edited lines 98-103 to what you copy and pasted and now I'm getting this error when sending a message to the bot: 

![screen shot 2015-07-29 at 1 51 46 pm](https://cloud.githubusercontent.com/assets/2515263/8969537/05b8b35a-35f9-11e5-9ef4-5cc1c920e067.png)

Any ideas?

**Comment by llimllib** [comment]
The fix for this was *finally* merged into master on Sep 23, in aac0e69a777632b377068d7677bb91678dbf79f8

So sorry for the delay!
